libraries/Bluefruit52Lib/src/services/BLEHidGeneric.cpp

```diff
@@ -260,6 +245,13 @@ err_t BLEHidGeneric::begin(void)
   VERIFY_STATUS( _chr_control.begin() );
   _chr_control.write8(0);
 
+  // HID Gamepad
```

Comment:
all hid profile should should be optional, the relation between hidGeneric and hidAdafruit is a bit ambiguous. My idea is allow advanced user to implement their own custom hid profile e.g ( fancy gamepad/joystick or any input device) by inherit the hidGeneric. While HidAdafruit simply implement the common basic profile (keyboard/mouse/gamepad) that cover most of user need. So yeah, I mean gamepad shouldn't be part of the Generic. Don't worry, I will pull the code and update this myself later on to wrap this up soon enough.